python/pylibcudf/pylibcudf/tests/test_nvtext_dedup.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.
+
+import pyarrow as pa
+import pytest
+from utils import assert_column_eq
+
+import pylibcudf as plc
+
+
+@pytest.fixture(scope="module")
+def input_col():
+    arr = [
+        "01234567890123456789",
+        "01234567890123456789",
+        "01234567890123456789",
+    ]
```

Comment:
Maybe make the construction clearer?
```suggestion
    arr = ["0123456789"] * 6
```
Then later `expected = "0123456789" * 5`?